doc/rtd/howto/ubuntu_test_prerelease.rst

```diff
@@ -0,0 +1,50 @@
+.. _ubuntu_test_pre_release:
+
+Test pre-release cloud-init
+===========================
+
+Afterthe cloud-init team creates an upstream release, cloud-init will
+be released in the -proposed APT repository for a
```

Comment:
>  Afaiu, -proposed is a pocket rather than an APT repository.

People say that word sometimes but I have no clue why they do and I've never seen it defined. The [ubuntu wiki](https://wiki.ubuntu.com/Testing/EnableProposed) uses the word repository, apt's docs on debian.org call it a [distribution](https://wiki.debian.org/SourcesList#sources.list_format) and the apt man page calls it a [suite](https://manpages.debian.org/bookworm/apt/sources.list.5.en.html).

If you have any clue which is most authoritative I'll happily change it to whichever is most correct. It seems to me though that there is disagreement about what to call it, and "pocket" is not one of the things I can actually find defined anywhere.

Reply:
A couple of pointers quasi-defining pockets and suites:

- https://github.com/canonical/ubuntu-maintainers-handbook/blob/a5a386be1be1507fbd1dd8842699b0758042169c/PackageTests.md?plain=1#L327
- https://help.launchpad.net/Glossary

As I understand it, -proposed represents apt suites and/or launchpad series + pocket depending on the context.

Reply:
Thanks for the definitions @aciba90. I wish those were easier to find than in some team-specific documentation, or in a miscellaneous glossary not related to APT.

I'm happy to go whatever we collectively think is best here, @s-makin and @aciba90. Thoughts?

Reply:
I guess that's on me for assuming that because I saw the terminology in the UMH it must have been standard packaging terminology that was defined somewhere officially :upside_down_face: even Debian hasn't really defined it. It feels wrong to refer to internal documentation for this sort of thing, so let's stick with the original terminology you used, or use suites since that's at least defined in the apt manpages so that if the users need to refer to the manpages, the terminology will at least be consistent (I definitely don't want to point them to the Ubuntu wiki).

Reply:
I want to make a stronger case for the "pocket" terminology. Repository is fairly generic term, and can refer to pretty much any place to get software. You could talk about the "Main" or "Universe" repository along with the "-updates" or "-proposed" repository, even though "Universe" and "proposed" aren't interchangeable. "Pocket" is more specific and lets you know exactly what is being referred to. You can't really find it in Debian docs (except the glossary) because it's ubuntu-specific.

Debian defines it here:
https://wiki.debian.org/Glossary#pocket

`The Archive Repository Pockets` section explains it well here:
https://ubuntu.com/blog/ubuntu-updates-releases-and-repositories-explained#The_archive_repository_pockets

Pocket is found 15 times on the SRU page, vs 1 for repository:
https://wiki.ubuntu.com/StableReleaseUpdates

There's more explanation of it here:
https://wiki.ubuntu.com/SecurityTeam/FAQ#Repositories_and_Updates

It's used in other ubuntu-specific tooling like here (search pocket):
https://manpages.ubuntu.com/manpages/jammy/man1/add-apt-repository.1.html

Since pocket is an established term that is more specific than repository, I think we should be preferring it.